features/image-transformations/overlay-using-layers.md

```diff
@@ -0,0 +1,200 @@
+
+## Differences between overlay text output in layer and non-layer syntax
+
+In overlay text, the output generated using the layers syntax may be slightly different from its non-layer syntax equivalent. If you are moving from non-layer overlay syntax to layer syntax, you may need to adjust some transformation parameter values to get a similar output as before.
+
+These differences include but are not limited to:
+
+- **Top and bottom padding:** Apparent empty space (or padding) on the top and bottom of overlay text will be slightly lesser in layer syntax than non-layer syntax. It can be handled by increasing the top and bottom padding accordingly using the `pa` layer transformation parameter.
+- **Trimming:** Any empty space around the text output will be trimmed by default in the layer syntax output. It can be handled by adding the padding manually using the `pa` layer transformation parameter.
+- **Line height:** The line height of the text will be marginally larger in layer syntax. It cannot be controlled manually for the moment.
+- **Line breakpoints:** The points at which sentences break when maximum width is used may be different. It cannot be controlled manually.
+- **Unicode language support:** Layer overlay text supports Unicode letters, marks, and numerals in other languages, while non-layer overlay text does not. 
+
+# Add solid color blocks over image
+You can add a solid color block over a base image using the following example.
+
+## Usage syntax
+
+```markup
+https://ik.imagekit.io/demo/base-image.jpg?tr=l-image,i-ik_canvas,bg-FF0000,w-300,h-100,l-end
+```
+
+You can also control the position of image overlay using these [positional parameters](overlay-using-layers#position-of-layer).
+
+## Transformation of solid color overlay
+
+Following transformation parameters are supported on the solid color background inside a layer.
+
+| Parameter                                                                                | Description                        |
+|------------------------------------------------------------------------------------------|------------------------------------|
+| w                                                                                        | Width of solid color background.   |
+| h                                                                                        | Height of solid color background.  |
+| [bg](../image-transformations/resize-crop-and-other-transformations#background-color-bg) | It is used to specify the solid color background color in RGB Hex Code (e.g. `FF0000`) or an RGBA Code (e.g. `FFAABB50`) or a color name (e.g. `red`). If you specify an 8 character background, the last two characters must be a number between `00` and `99`, which indicate the opacity level of the background. `00` represents an opacity level of `0.00`, `01` represents opacity level `0.01`, and so on. |
```

Comment:
> used to specify the solid color background color

just 'used to specify the solid color' would be better?

Reply:
Done